### DISCUSSION TITLE
After stopping a 3d print, move the head to the parking spot.

### DISCUSSION THREAD

**Discussion by heidepiek** [mriscoc/Ender3V2S1#1292] [Q&A]
It sometimes happens that I have to stop a print  (mid print). Not pause, but really stop. When I press stop, the print head stays where it was then.  Is it also possible to send the print head to its parking position after stopping a print? 

Harrie

**Comment by mriscoc** [comment]
The stop button is not only used as a Cancel print operation, but also as an emergency stop, so in the last use it is not convenient to move the print head. You can manually park the head by using Prepare/Filament management menu.

**Reply by heidepiek** [reply]
I know when i pause the print in the home screen it goes to the park place. But than you have 2 options, purge and resume. You can`t stop the print. Sometimes if  i see that the first layer is not good, or there is a peace of fillament stuck on a place i don`t want it. I sometimes stop the print. And start a new one. But if you stop, it really stops (like you said emergency stop) if i cancel you have too resume after pause and stil have to use stop print. I would like a cancel button that stops the print and goes to a place where i wanted him to go after a stop. Hope i make sense miguel. I now know that the stop button is also used as an emergency stop. Did not know that.

Harrie
